packages/shorebird_tests/test/shorebird_tests.dart

```diff
@@ -0,0 +1,191 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:archive/archive_io.dart';
+import 'package:path/path.dart' as path;
+
+import 'package:meta/meta.dart';
+import 'package:test/test.dart';
+
+File get _flutterBinaryFile => File(
```

Comment:
Doc comments throughout this file would be useful. Here: "this assumes we're running from (path)" or similar